src/lib/_imports/trumps/_s-portal-nav/_s-portal-nav.hbs

```diff
@@ -8,19 +8,29 @@
 -->
 <ul class="s-portal-nav navbar-nav">
   <!-- When user is not logged in, there are zero dropdown(-*) classes -->
+  <style type="text/css">
+    .s-portal-nav .icon {
+      margin-right: 0.25em;
+
+      /* Copied from FontAwesome `.fa-lg` */
+      font-size: 1.3em;
+      vertical-align: text-bottom;
+      /* tweaked to align better */
+    }
+
+    .s-portal-nav .nav-link .icon {
+      font-size: 1.5em;
+      vertical-align: middle;
+      /* tweaked to align better */
+    }
+  </style>
```

Comment:
We found these styles to apparently be necessary. But they are messy (my fault, unmaintained). Please move them to `s-portal-nav.css` and use the same `font-size` and `vertical-align` for both icons (whichever value makes the icon size and alignment look good). That should also allow you to reduce the amount of code. No need to retain the comments.